File: samples_generated_code/sample_3/qwen/baseline_code_c/code_row_53.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <time.h>

int main() {
    srand(time(NULL));

    float random1 = (float)rand() / RAND_MAX;
    float random2 = (float)rand() / RAND_MAX;
    float random3 = (float)rand() / RAND_MAX;

    char str1[50], str2[50], str3[50];
    sprintf(str1, "%.4f", random1);
    sprintf(str2, "%.4f", random2);
    sprintf(str3, "%.4f", random3);

    char result[150] = "";
    strcat(result, str1);
    strcat(result, str2);
    strcat(result, str3);

    FILE *file = fopen("output.txt", "w");
    if (file == NULL) {
        printf("Error opening file!\n");
        return 1;
    }
    fprintf(file, "%s", result);
    fclose(file);

    return 0;
}

File: samples_generated_code/sample_1/athene/baseline_code_java/code_row_53.java
import java.io.FileWriter;
import java.io.IOException;
import java.util.Random;

public class RandomFloatsToFile {
    public static void main(String[] args) {
        Random rand = new Random();
        float f1 = rand.nextFloat();
        float f2 = rand.nextFloat();
        float f3 = rand.nextFloat();

        String str1 = Float.toString(f1);
        String str2 = Float.toString(f2);
        String str3 = Float.toString(f3);

        String result = str1 + str2 + str3;

        try (FileWriter fw = new FileWriter("output.txt")) {
            fw.write(result);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}